.github/workflows/upload-game-installers.yml

```diff
@@ -25,8 +25,6 @@ jobs:
         with:
           distribution: temurin
           java-version: 11
-      - name: Run Build Checks
-        run: ./game-app/run/check
```

Comment:
`./game-app/run/check` runs a super fancy version of 'gradlew check' & also a custom 'check-links' script. Another PR will make that check its own github action.